hack/unwanted-dependencies.json

```diff
@@ -1,7 +1,11 @@
 {
   "spec": {
     "unwantedModules": {
```

Comment:
added this to capture current state to make sure we don't add more links to these between now and when genproto drops backreferences and we can clean these up